src/main/java/org/railway/BookTicketPage.java

```diff
@@ -0,0 +1,48 @@
+package org.railway;
+
+import org.openqa.selenium.By;
+import org.railway.utils.Action;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+public class BookTicketPage extends BasePage{
+
+    private String dateDate = "//select[@name='Date']//option[text()='%s']";
+    private By bookTicketbutton = By.xpath("//input[@value='Book ticket']");
+    //private By bookTicketTable = By.xpath("//table[@class='MyTable WideTable']");
+    private String selectOption = "//select[@name='%s']";
+
+
+    public void selectDepartdate(int date){
+        Calendar calendar = Calendar.getInstance();
+        calendar.add(Calendar.DAY_OF_YEAR, date);
```

Comment:
for handle date, it's better if separate it into another class (e/x: 'DateUtils')